lms/static/sass/discussion/elements/_navigation.scss

```diff
@@ -14,15 +14,15 @@
 // Header
 // ------
 .forum-nav-header {
-  @include box-sizing(border-box);
+  box-sizing: border-box;
   display: table;
```

Comment:
Interesting. There are 40 usages of ``display: table`` and ``display: table-cell`` in the LMS's sass. We'll have to clean these up as move over to the Pattern Library.